docs/hub/models-cards-co2.md

```diff
@@ -20,11 +20,11 @@ You can add the carbon footprint data to the model card metadata (in the README.
 ```yaml
 ---
 co2_eq_emissions:
-      emissions: "in grams of CO2"
-      source: "source of the information, either directly from AutoTrain, code carbon or from a scientific article documenting the model"
-      training_type: "pre-training or fine-tuning"
-      geographical_location: "as granular as possible, for instance Quebec, Canada or Brooklyn, NY, USA"
-      hardware_used: "how much compute and what kind, e.g. 8 v100 GPUs"
+  emissions: "in grams of CO2"
+  source: "source of the information, either directly from AutoTrain, code carbon or from a scientific article documenting the model"
```

Comment:
maybe a link to Code Carbon here, i.e.:
source: "source of the information, either directly from AutoTrain, [Code Carbon](https://github.com/mlco2/codecarbon) or from a scientific article documenting the model"

Reply:
i think this is mentionned in the same page just above, no?

> the estimated carbon footprint of the model, calculated in real-time with the [Code Carbon](https://github.com/mlco2/codecarbon) package or after training using the [ML CO2 Calculator](https://mlco2.github.io/impact/).